### ISSUE TITLE
Specific question about formatting block quotes

### ISSUE THREAD

**Issue by blankens10** [lowercasename/docdown#25]
I've customized my reference.docx file to correctly format block quotes, except for one aspect: block quotes are ordinarily followed by commentary that is considered part of the paragraph started above the block quote. Is there any way to get DocDown to not indent body text immediately after a block quote?

I tried what would seem like the obvious solution using the styles pane, but to no avail. the only styles I use are normal, block quote, code, and footnote, plus 2 levels of headings. is there some other style preset supported by pandoc that I could repurpose to make this happen?

Any help appreciated.

**Comment by lowercasename** [comment]
Ah, I wish! No, Markdown and Pandoc (which is the software DocDown uses under the hood) will only know to distinguish 'paragraph' from 'blockquote', so there's no way to do that sadly. You could, however, set Word's 'Style for following paragraph' option in the style editor to a version of the standard paragraph style without an indent. That would only work 100% of the time if none of your blockquotes finish paragraphs, or if you leave double line breaks between paragraphs!

**Comment by blankens10** [comment]
Well, I did that "style for following paragraph" to "body 2" and set to not have indents, didn't work. Pandocs markdown requires an empty line to get out of the block quote, and that appears to trigger "paragraph" style again, regardless of the style settings.

OTOH, for a handle bit, yours is pretty solid.

**Comment by mcvmcvmcvmcv** [comment]
hi all, i was just looking into this myself. it seems that in order to trigger the "Style for following paragraph" you have to physically type a carriage return. for now i am just planning to use the styles pane to select all, then highlight my blockquote text, scroll through and use the 'no indent' style that i made to get them all in line. a pain, but for all the other great things that docdown enables, not a huge deal i think!

**Comment by blankens10** [comment]
I figured out one way for doing this. I built the app from the source code, but I found the two instances of 'docx' as a command line argument in main.js and changed them to 'docx+styles'. I formatted the reference.docx so that 'first paragraph' is not indented, and 'body text' is, and then you just have to put a style block ::: {custom-style="Body Text"} <text>  ::: around the first paragraph after a heading, and voila, all the paragraphs following block quotes are automatically not indented, but everything else is. 

Just thought I'd share.
